src/types/textual.md

```diff
@@ -17,7 +17,9 @@ is valid UTF-8. Calling a `str` method with a non-UTF-8 buffer can cause
 Since `str` is a [dynamically sized type], it can only be instantiated through a
 pointer type, such as `&str`.
 
-## Bit validity
+## Layout and bit validity
+
+`char` is guaranteed to have the same size and alignment as `u32` on all platforms.
```

Comment:
```suggestion
`char` is guaranteed to have the same size and alignment as `u32` on all platforms.

`Option<char>` is guaranteed to have the same size and alignment as `char` on all platforms.
```

(Unless we want to put this somewhere else.)

Reply:
This suggestion should probably be handled separately, not in this PR.

Reply:
Filed https://github.com/rust-lang/reference/issues/1426 to track this.